### PR TITLE
#87 - implemented I18n localization for android and iOS

### PR DESCRIPTION
**Pull request by akshaydoshi2**
/claim #87 
I18n localization for android and iOS has been implemented.

## For Android:
1. The key-value pairs for localizations can be found in res/values-<language_code>/strings.xml
2. The file res/values/strings.xml is the fallback file. If the user selects any language that is not implemented by us, the values from this file are used.
3. These strings can be used in the app using Resources.getSystem().getString(R.string.<name_property>). For eg: Resources.getSystem().getString(R.string.hello_world)
4. New languages can be added by simple creating new folder like in: res/values-<language_code>/strings.xml and inserting all the translated strings.
## For iOS:
1. The key-value pairs for localizations can be found in Localizable in the left navigation pane.
2. Each language uses a separate file named as Localizable (language) in the left navigation pane.
3. The strings can be accessed by: NSLocalizedString(<key_name>, comment:""). For example: NSLocalizedString("hello_world", comment:"").
4. New languages can be added by doing the following in XCode: `Plugin (in the left navigation pane) -> Plugin (Project) -> Info -> Localizations -> Press (+) -> Select language

_For both android and iOS, the keys of all the translated matching strings should match. Else the translations will not work._

### CODE REVIEW

File: android/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+<!--The values in this file are the fallback values in case the language selected by the user is not available here.
+The values of the name property should match in all the strings files
+The strings can be accessed by: Resources.getSystem().getString(R.string.<name_property>)
+ For example: Resources.getSystem().getString(R.string.hello_world)-->
+
+<resources>
+    <string name="hello_world">Hello World</string>
+    <string name="success">success</string>
+    <string name="failed">fallido</string>
```
Comment (wesleymatosdev) [inline android/src/main/res/values/strings.xml]: Can you change this to English?

File: ios/Plugin/CapacitorUpdaterPlugin.swift [hunk 2]
```diff
@@ -181,7 +181,7 @@ public class CapacitorUpdaterPlugin: CAPPlugin {
                 self.notifyListeners("downloadFailed", data: ["version": version])
                 let current: BundleInfo = self.implementation.getCurrentBundle()
                 self.implementation.sendStats(action: "download_fail", versionName: current.getVersionName())
-                call.reject("Failed to download from: \(url!)", error.localizedDescription)
+                call.reject(NSLocalizedString("failedToDownload", comment: "")+url?.absoluteString, error.localizedDescription)
```
Comment (wesleymatosdev) [inline ios/Plugin/CapacitorUpdaterPlugin.swift]:
```suggestion
                call.reject(NSLocalizedString("failedToDownload", comment: "") + url?.absoluteString, error.localizedDescription)
```

**Review by ologbonowiwi — APPROVED** [review]
LGTM

File: android/src/main/res/values/strings.xml (hunk 1 above)
Comment (akshaydoshi2) [reply android/src/main/res/values/strings.xml]: Hey, should I push this change and create a new pull request? The `failed` string is not being used any where right now.

Comment (wesleymatosdev) [reply android/src/main/res/values/strings.xml]: in this case we can remove it

Comment (akshaydoshi2) [reply android/src/main/res/values/strings.xml]: Hey, I have corrected it.

**Comment by akshay-virtoustack** [comment]
@riderx can you check this?

**Comment by akshaydoshi2** [comment]
@riderx can you check this please?

**Pull request closed without merging**